src/app/devices/deviceContent/components/digitalTwin/digitalTwinInterfaces.tsx

```diff
@@ -35,36 +53,36 @@ interface ModelContent {
 }
 
 export const DigitalTwinInterfaces: React.FC<DigitalTwinInterfacesProps> = props => {
-    if (props.isLoading) {
-        return <MultiLineShimmer/>;
-    }
-
     const url = props.match.url;
     const deviceId = getDeviceIdFromQueryString(props);
     const { t } = useLocalizationContext();
+    const { modelId, componentNameToIds, retrieveComponents, modelDefinitionWithSource } = props;
+
+    React.useEffect(() => {
+        if (!modelId) {
```

Comment:
in which case you will not have a model id?

Reply:
it normally wouldn't, as this page in most cases would not be rendered if modelId does not exist. It only exist when users was on this page, the service went down and user hit refresh.